packages/netlify-cms-backend-github/src/API.js

```diff
@@ -4,16 +4,22 @@ import { uniq, initial, last, get, find, hasIn, partial, result } from 'lodash';
 import { filterPromises, resolvePromiseProperties } from 'netlify-cms-lib-util';
 import { APIError, EditorialWorkflowError } from 'netlify-cms-lib-util';
 
-const CMS_BRANCH_PREFIX = 'cms/';
-
 export default class API {
   constructor(config) {
     this.api_root = config.api_root || 'https://api.github.com';
     this.token = config.token || false;
-    this.branch = config.branch || 'master';
-    this.repo = config.repo || '';
-    this.repoURL = `/repos/${this.repo}`;
-    this.merge_method = config.squash_merges ? 'squash' : 'merge';
+    this.branch = config.branch || "master";
+    if (config.cms_branch_prefix) {
```

Comment:
The `config` argument here is _not_ the value of the `config.yml`, it's the argument passed in here: https://github.com/netlify/netlify-cms/blob/master/packages/netlify-cms-backend-github/src/implementation.js#L45-L52. Thus, this isn't actually going to work, though it's very close.

Additionally, I would suggest we use `backend.branch_prefix` instead of just `cms_branch_prefix` at the top level of the config.

Reply:
Yes I see. I need to fix that I had passed in something else. Fixing.
I like "branch_prefix" as the name. Changing.